bin/release.py

```diff
@@ -0,0 +1,124 @@
+"""A script which triggers and wait for an azure devops to complete a release"""
+from ast import parse
+import os
+import sys
+import time
+from azure.devops.connection import Connection
+from msrest.authentication import BasicAuthentication
+
+FAILED_STATUSES = ["canceled", "partiallySucceeded", "rejected"]
+COMPLETED_STATUSES = ["succeeded", "canceled", "partiallySucceeded", "rejected"]
+
+
+def create_ado_connection(organization, ADO_PAT) -> Connection:
+    """Returns an ADO connection to call the ADO Rest APIs."""
+
+    organization_url = "https://dev.azure.com/{0}".format(organization)
+    credentials = BasicAuthentication("", ADO_PAT)

```

Comment:
How long does this PAT live?

Reply:
We can define its lifetime until one year.

Reply:
How does that adhere to our security policies? 1 years is TOO long!

Reply:
Sorry I was not clear. I meant anywhere between a day to one year. It's upto us to define the lifetime between the range.